pkg/cluster/operation/destroy.go

```diff
@@ -91,6 +90,46 @@ func Destroy(
 	return nil
 }
 
+// StopAndDestroyInstance stop and destroy the instance,
+// if this instance is the host's last one, and the host has monitor deployed,
+// we need to destroy the monitor, either
+func StopAndDestroyInstance(getter ExecutorGetter, component spec.Component, cluster spec.Topology, instance spec.Instance, options Options, destroyMonitor bool) error {
```

Comment:
I think you can remove the `component` arg and use `instance.ComponentName()` or `instance.ID()` instead.

Reply:
This arg is reserved, maybe has other usage later

Reply:
Then what about add it back when we really need it?

Reply:
LGTM